asab/library/service.py

```diff
@@ -514,7 +505,74 @@ def on_library_change(self, message, provider, path):
 		if isinstance(paths, str):
 			paths = [paths]
 		for path in paths:
-			assert path[:1] == '/', "Absolute path must be used when subscribing to the library changes"
+			if not path.startswith("/"):
+				raise LibraryInvalidPathError(
+					message="Directory path must start with '/' when subscribing to Library changes.",
+					path=path,
+				)
 
 			for provider in self.Libraries:
 				await provider.subscribe(path)
+
+
+def _validate_path_item(path: str) -> None:
+	# File path must start with '/'
+	if not path.startswith("/"):
+		raise LibraryInvalidPathError(
+			message="Item path must start with '/' (e.g. '/Templates/item.json')",
+			path=path,
+		)
+
+	# File path must end with extension (e.g. '.json')
+	if not len(os.path.splitext(path)[1]) > 0:
```

Comment:
We have a more specific condition for extension regarding a length of that.
Consider using that.